eng/source-build-release-official.yml

```diff
@@ -62,9 +62,13 @@ parameters:
   type: boolean
   default: true
 - name: createReleaseAnnouncement
-  displayName: Create Release Announcement
+  displayName: Create release announcement
```

Comment:
FYI, we are rather inconsistent on the displayName casing across the board.  Most appear to use Title casing - Dyn Run, Submit Release PR, etc.